### DISCUSSION TITLE
về SPRPINT 3

### DISCUSSION THREAD

**Discussion by dinhgiaan** [dinhgiaandev/CodeGuru_BE#46] [General]
- mấy fen thấy sprint 3 chia vậy có ổn không? hay có cần thêm gì vô không?

**Comment by ankieu123** [comment]
mình nên thêm vài chức năng quản trị viên mới cho đề tài của mình (●'◡'●)

**Reply by dinhgiaan** [reply]
cũng được, nhưng mà giờ phần tạo khóa học code mất thời gian lắm, sợ k có đủ thời gian đâu, chắc để qua sprint 4 đi
